content/hi/docs/home/_index.md

```diff
@@ -43,22 +43,22 @@ cards:
   title: "प्रशिक्षण"
   description: "कुबेरनेट्स में प्रमाणित हो कर अपने क्लाउड नेटिव प्रोजेक्ट सफल बनाएं!"
   button: "प्रशिक्षण देखें"
-  button_path: "/training"
+  button_path:/docs/reference
```

Comment:
Looks like there is missing space in this line, it is causing the build to fail with the following error:
[(Build logs)](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/659ea3b9f35337000810e603)

> 2:04:11 PM: Failed during stage "building site": Build script returned non-zero exit code: 2
> 2:04:11 PM: Total in 2625 ms
> 2:04:11 PM: Error: error building site: assemble: "/opt/build/repo/content/hi/docs/home/_index.md:46:1": failed to unmarshal YAML: yaml: line 46: could not find expected ":"

Try adding space and check if that helps with this error.

```suggestion
  button_path: /docs/reference
```